auto_submit/lib/requests/github_webhook.dart

```diff
@@ -33,9 +41,14 @@ class GithubWebhook extends RequestHandler {
     hasAutosubmit = pullRequest.labels!.any((label) => label.name == 'autosubmit');
 
     if (hasAutosubmit) {
-      // TODO(kristinbi): Check if PR can be submitted. https://github.com/flutter/flutter/issues/98707
-    }
+      final String githubToken = Platform.environment['GITHUB_TOKEN']!;
+      final GithubService gitHub = config.createGithubServiceWithToken(githubToken);
+
+      final RepositorySlug slug = RepositorySlug.full(body['repository']['full_name']);
+      final int number = body['number'];
```

Comment:
These are not used. You can either comment them out, skip them, or use log to have some dummy use of them.

Reply:
Sure, I will comment them out in this pr to pass the check.